lambdas/service/app.py

```diff
@@ -924,6 +924,11 @@ def handle_manifest_generation_request():
     token = query_params.get('token')
     retry_url = self_url()
     manifest_service = AsyncManifestService()
+    if format_ == 'full':
```

Comment:
This assumes that the generator for the full manifest is cacheable. In general, you shouldn't make decision based on the format string anymore. The format is used once to determine the generator implementation and then that implementation is interrogated about it's capabilities.